docs/working-with-kyverno.md

```diff
@@ -0,0 +1,314 @@
+             requests:
+               cpu: 100m
+               memory: 128Mi
+           securityContext:
+             allowPrivilegeEscalation: false
+             capabilities:
+               drop:
+               - ALL
+             privileged: false
+             readOnlyRootFilesystem: true
+             runAsNonRoot: true
+           volumeMounts:
+           - mountPath: /.sigstore
+             name: sigstore
+           - mountPath: /etc/kubeconfig
+             name: kubeconfig
+             subPath: kubeconfig
+         initContainers:
+           - args:
+         - env:
+           - name: METRICS_CONFIG
+             value: kyverno-metrics
+           - name: KYVERNO_NAMESPACE
+             valueFrom:
+               fieldRef:
+                 fieldPath: metadata.namespace
+           image: ghcr.io/kyverno/kyvernopre:latest
+           imagePullPolicy: Always
+           name: kyverno-pre
+           resources:
+             limits:
+               cpu: 100m
+               memory: 256Mi
+             requests:
+               cpu: 10m
+               memory: 64Mi
+           securityContext:
+             allowPrivilegeEscalation: false
+             capabilities:
+               drop:
+               - ALL
+             privileged: false
+             readOnlyRootFilesystem: true
+             runAsNonRoot: true
+         securityContext:
+           runAsNonRoot: true
+         serviceAccountName: kyverno-service-account
+         volumes:
+         - emptyDir: {}
+           name: sigstore
+         - name: kubeconfig
+           secret:
+              defaultMode: 420
+              secretName: kubeconfig
+   ---
+   apiVersion: v1
+      stringData:
+         kubeconfig: |-
+            apiVersion: v1
+            clusters:
+            - cluster:
+                certificate-authority-data: {{ca_crt}}
+                server: https://karmada-apiserver.karmada-system.svc.cluster.local:5443
+              name: kind-karmada
+            contexts:
+            - context:
+                cluster: kind-karmada
+                user: kind-karmada
+              name: karmada
+            current-context: karmada
+            kind: Config
+            preferences: {}
+            users:
+            - name: kind-karmada
+              user:
+                client-certificate-data: {{client_cer}}
+                client-key-data: {{client_key}}
+      kind: Secret
+      metadata:
+         name: kubeconfig
+         namespace: kyverno
+   ```
+
+   For multi-cluster deployment, We need to add the config of `--serverIP` which is the address of the webhook server. So we need to ensure that the network from node in karmada control plane to those in karmada-host cluster is connected and expose kyverno controller pods to control plane, for example, using `nodePort` above. Then, fill in the secret which represents kubeconfig pointing to karmada-apiserver.
+
+## Run demo
+### Create require-labels clusterpolicy
```

Comment:
Please add a short description about the `ClusterPolicy`, such as what it is and what it is used for.